Sources/Brave/Frontend/Browser/Tab.swift

```diff
@@ -269,14 +295,14 @@ class Tab: NSObject {
     self.id = id
     rewardsId = UInt32.random(in: 1...UInt32.max)
     nightMode = Preferences.General.nightModeEnabled.value
-    syncTab = tabGeneratorAPI?.createBraveSyncTab(isOffTheRecord: type == .private)
+    _syncTab = tabGeneratorAPI?.createBraveSyncTab(isOffTheRecord: type == .private)
     
-    if let syncTab = syncTab {
-      faviconDriver = FaviconDriver(webState: syncTab.webState).then {
+    if let syncTab = _syncTab {
```

Comment:
I don't think matters here but for consistency should this access remain on `syncTab` (instead of `_syncTab`) to match others?

Reply:
It shouldn't make a difference since this is done in the constructor, and the driver only cares about the internal `webState` which we don't control from the Swift side.